### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by 1fabi0** [jlennox/LibYuvSharp#2]
I really appreciate this wrapper, but since 2019 there were added a few new Methods, like NV12Scale to the LibYuv.

**Comment by jlennox** [comment]
@1fabi0 Would you be interested in adding support for them?

**Comment by 1fabi0** [comment]
Yes if you could upload it to nuget after I did the PR I would be very thankful

**Comment by jlennox** [comment]
Yes that works for me.

I tried to build the latest libyuv and I'm hitting this during the sync process:

```

[0:38:43] Still working on:
[0:38:43]   src/third_party/icu
Syncing projects: 100% (22/22), done.
Username for 'https://chrome-internal.googlesource.com':
```

Not sure how required that third party lib is :/

edit: seems the bypass to that is `set DEPOT_TOOLS_WIN_TOOLCHAIN=0`

**Comment by 1fabi0** [comment]
I had the same problem, I also had to install vs 2019 because 2022 does not work with libyuv ... but i was able to build it and update the library #3

**Comment by freshman-lwl** [comment]
I use it in xamarin, and when I run it, it shows that the dll is not found...But it prompts an error “MSB3270”. The error is gone after setting as "warn", but it doesn't work.

**Comment by 1fabi0** [comment]
The library is not included as build for android or ios... Thats the issue

**Comment by jlennox** [comment]
There's only a Windows x64 build included. I do not have the means to cross-compile for multiple targets unfortunately.
